e2-studio-star-rx72n-firmware/libs/rx_spi_link/src/rx_spi_link.c

```
@@ -0,0 +1,629 @@
+    return k_rx_err_retry_limit;
+}
+
+/* =============================================================================
+ * Receive API Implementation
+ * ============================================================================= */
+
+/**
+ * @brief Receive and decode a frame from SPI with HARQ+FEC
+ * @see rx_spi_link.h for full documentation
+ */
+rx_err_t rx_spi_link_receive(rx_spi_link_t*                link,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_spi_link_receive' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)